engine/src/w32dcw32.cpp

```diff
@@ -835,7 +835,7 @@ LRESULT CALLBACK MCWindowProc(HWND hwnd, UINT msg, WPARAM wParam,
 				// Pressing Alt and the key "+" starts a number-typing sequence.
 				//  Otherwise, we are not in such a sequence - be it because a normal
 				//  char has been typed, or because the sequence is terminated.
```

Comment:
Could you also change this comment to reflect the fact that the + key is not needed?

Reply:
Sure :)